tests/test_generators.py

```diff
@@ -9,6 +11,15 @@
 from populous import generators
 from populous.generators.base import NullableMixin
 from populous.generators.base import UniquenessMixin
+from populous.generators.date import to_timestamp
+
+
+@contextmanager
+def not_raises(exception):
+    try:
+        yield
+    except exception:
+        raise pytest.fail("DID RAISE {0}".format(exception))
```

Comment:
this is a funny to way to test for the absence of failure :) . It would work as well without this context manager, but it is indeed more explicit that way :) .